### ISSUE TITLE
No module for creating CIFS local user

### ISSUE THREAD

**Issue by jalon-kouryuu** [ansible-collections/netapp.ontap#27]
### Summary

I am trying to use netapp.ontap connection to automatize the process of storage setup. However, I find that there is no module for creating a new CIFS local user (maybe just I don't know). It will be very helpful if you can provide this module for us as soon as possible. Thanks a lot.

### Component Name
na_ontap_cifs_local_user

### Additional Information

```yaml (paste below)
- name: Create local CIFS local user
      netapp.ontap.na_ontap_cifs_local_user:
       state: present
        name: test_user
        set_password:  test_password
        vserver: vssample
        username: '{{ username }}'
        password: '{{ password }}'
        hostname: '{{ hostname }}'
        https: true
        validate_certs: false
```

**Comment by carchi8py** [comment]
We have created an internal story for this DEVOPS-4869

**Comment by lonico** [comment]
fixed in 22.2.0
